docs/CONNECTOR_PROTOCOL.md

```diff
@@ -114,13 +114,14 @@ This is our main communication index, used to communicate the connector's config
   index_name: string;   -> The name of the content index where data will be written to
   is_native: boolean;   -> Whether this is a native connector
   language: string;     -> the language used for the analyzer
+  last_access_control_sync_error: keyword;   -> Optional last access control sync job error message
```

Comment:
```suggestion
  last_access_control_sync_error: string;   -> Optional last access control sync job error message
```

Reply:
Enterprise Search actually uses `last_sync_error: { type: 'keyword' }` as mapping for `last_sync_error`, so I simply used the same type to be consistent. So I guess our docs are wrong or we need to change the mapping for `last_sync_error` and `last_access_control_sync_error`.

Reply:
> Enterprise Search actually uses `last_sync_error: { type: 'keyword' }` as mapping for `last_sync_error`

Noticed this as well: https://github.com/elastic/connectors-python/pull/1011#pullrequestreview-1464065947

But the suggestion above is not the exact data type. `last_sync_error` also uses `string`.
https://github.com/elastic/connectors-python/blob/013b0ada6301ed92fef1cbab793d31c24e7a01c7/docs/CONNECTOR_PROTOCOL.md?plain=1#L123